sw/device/lib/testing/hmac_testutils.h

```diff
@@ -11,10 +11,91 @@
 #include "sw/device/lib/dif/dif_hmac.h"
 
 /**
- * Read and compare the length of the message in the HMAC engine to the length
- * of the message sent in bits.
+ * Timeouts to be used for different HMAC operations.
+ *
+ * All timeouts are calculated assuming a 100kHz clock frequency, in order
+ * to cover a range of targets. Please see:
+ * https://docs.opentitan.org/hw/ip/hmac/doc/
+ */
+typedef enum {
+  /**
+   * FIFO empty 10uS timeout.
+   *
+   * single HMAC block compression takes 80 cycles.
+   */
+  kHmacTestutilsFifoEmptyTimeoutUsec = 10,
+  /**
+   * HMAC done 50uS timeout.
+   *
+   * Final hash calculation takes 360 cycles, which consists of one block
+   * compression and extra HMAC computation.
+   */
+  kHmacTestutilsFinishTimeoutUsec = 50,
+} hmac_testutils_timeout_t;
+
+/**
+ * Reads and compares the actual length sent message length against expected.
```

Comment:
I think there's a `length` in excess here.

Reply:
Sorry, I think I was not clear enough.
```suggestion
 * Reads and compares the actual sent message length against expected.
```

Reply:
Done.